pkg/v1/sdk/capabilities/discovery/tkg/capabilities.go

```diff
@@ -38,7 +39,9 @@ func (dc *DiscoveryClient) IsTKGm(ctx context.Context) (bool, error) {
 
 // IsTKGS returns true if the cluster is a TKGS cluster.
 func (dc *DiscoveryClient) IsTKGS(ctx context.Context) (bool, error) {
-	return dc.HasTanzuKubernetesClusterV1alpha1(ctx)
```

Comment:
Maybe a comment here that says this looks for any version of TKC API?

Reply:
Added comment.